app/queries/keyset_pagination.rb

```diff
@@ -0,0 +1,69 @@
+module Queries
+  class KeysetPagination
+    attr_reader :query, :order, :key, :count, :page
+
+    def initialize(query, key: nil, order: nil, count: nil, page:)
+      @query = query
+      @key = key || DEFAULT_KEY
+      @order = order || DEFAULT_ORDER
+      @count = (count || DEFAULT_COUNT).to_i
+      @page = page
```

Comment:
I wonder if these need to be constants? 

In trying to understand this I've struggled with what we might expect these should be in terms of type format, and that might help.

As this is meant to be something that can be reused might be good to add some param documentation to this class too.